packages/js/product-editor/src/blocks/images/edit.tsx

```diff
@@ -87,6 +87,11 @@ export function Edit() {
 						multipleSelect={ true }
 						onError={ () => null }
 						onFileUploadChange={ onFileUpload }
+						onMediaGalleryOpen={ () => {
+							recordEvent(
+								'product_images_media_gallery_open'
```

Comment:
I think that the proposed name was `wcadmin_product_add_image_button`, however `wcadmin_product_images_media_gallery_open` is more specific.
Pinging @pmcpinto to let him know that we renamed the event.